src/services/token.service.js

```diff
@@ -0,0 +1,27 @@
+import { Token } from '../models/token.js';
+
+async function save(userId, newToken) {
+  const token = await Token.findOne({ where: { userId } });
+
+  if (!token) {
+    await Token.create({ userId, refreshToken: newToken });
+    return;
+  }
+
+  token.refreshToken = newToken;
+  await token.save();
+}
+
+function getByToken(refreshToken) {
+  return Token.findOne({ where: { refreshToken } });
+}
+
+function remove(userId) {
+  Token.destroy({ where: { userId } });
```

Comment:
Consider using `await` with `Token.destroy` to ensure the operation completes before any subsequent logic is executed. This is important for consistency and to handle any potential errors properly.